src/Microsoft.AspNetCore.Http.Connections.Client/Internal/LongPollingTransport.cs

```diff
@@ -49,6 +49,14 @@ public Task StartAsync(Uri url, TransferFormat transferFormat)
 
             Log.StartTransport(_logger, transferFormat);
 
+
+            // Do the initial poll, this will return immediately without data
```

Comment:
What happens if in the short time between the connection starting and this first "poll" a message is written?

Reply:
What do you mean? The pipe will have the data and it’ll work like normal.

Reply:
Doesn't the poll response need to be written to the application like `Poll` does?

Reply:
Ok. I think the terminology needs to be cleaned up a little because the first request when using long polling isn't really a poll because it will never return data, it's more like it is used to initialize polling.

Splitting hairs but could you update the comment to say something like

```
// Make initial long polling request
// Server side implementation of long polling will always return the first request immediately and
// without data so response content can be ignored
```

Reply:
Sure.

Reply:
https://github.com/SignalR/SignalR/blob/1fba14fa3437e24c204dfaf8a18db3fce8acad3c/src/Microsoft.AspNet.SignalR.Core/PersistentConnection.cs#L32

Reply:
Throw back 😄